shell/platform/darwin/ios/framework/Source/IOKit.h

```diff
@@ -0,0 +1,229 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Has this file been copied from somewhere? If so could you add a comment pointing to the source or reference? Reading it seems like it's an accumulation of a bunch of IOKit headers into one file.

Reply:
Yep, added documentation.

Reply:
How does copyright work when the header is an amalgamation of other (presumably copyrighted) headers?